language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/ChangeVariableTypeCodeAction.java

```diff
@@ -206,6 +208,9 @@ private Optional<Node> getTypeNode(Node matchedNode, CodeActionContext context)
                 return Optional.ofNullable(constDecl.typeDescriptor().orElse(null));
             case OBJECT_FIELD:
                 return Optional.of(((ObjectFieldNode) matchedNode).typeName());
+            case LET_VAR_DECL:
+                LetVariableDeclarationNode variableDecl = (LetVariableDeclarationNode) matchedNode;
+                return Optional.ofNullable(variableDecl.typedBindingPattern().typeDescriptor());
```

Comment:
Can directly cast `matchedNode` here since this is the only usage of `variableDecl`